js/index.js

```diff
@@ -0,0 +1,86 @@
+jQuery(document).ready(function() {
+	function close_accordion_section() {
+		$('.accordion .accordion-section-title').removeClass('active');
+		$('.accordion .accordion-section-content').slideUp(300).removeClass('open');
+	}
+
+		$('.accordion-section-title').click(function(event) {
+		event.preventDefault();
+
+		// Grabs current anchor value
+		var currentAttrValue = $(this).attr('href');
+
+		if($(event.target).is('.active')) {
+			close_accordion_section();
+		}else {
+			close_accordion_section();
```

Comment:
as a resut, you could put the line outside the if/else block, and make your code that bit simpler.